src/Microsoft.Framework.ApplicationHost2/project.json

```diff
@@ -51,12 +55,12 @@
                 "System.Collections": "4.0.10-beta-*",
                 "System.Diagnostics.Tools": "4.0.0-beta-*",
                 "System.Diagnostics.Debug": "4.0.10-beta-*",
+                "System.Diagnostics.TraceSource": "4.0.0-beta-*",
```

Comment:
Ugh, noooo. We need to decoupled from the logging implementation and write our own.